book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -438,22 +597,27 @@ The following tags can be used for state data assimilation. More detailed inform
 * **end.date** : [required?] end date of the state data assimilation (in YYYY/MM/DD format)
 * **_NOTE:_** start.date and end.date are distinct from values set in the run tag because this analysis can be done over a subset of the run.
 
-## Parameter Data Assimilation
+### (experimental) Brown Dog {#xml-browndog}
 
-The following tags can be used for state data assimilation. More detailed information can be found here: [Parameter Data Assimilation Documentation](pda.documentation.md)
+This section describes how to connect to [Brown Dog](http://browndog.ncsa.illinois.edu). This facilitates processing and conversions of data (currently only meteorological input data).
```

Comment:
Drop `(currently only meteorological input data)` as this is in `load_data`

Reply:
Addressed in 3e143f954.